appyters/RNA_Level_Functional_Predictions/templates/fields/AutocompleteField.j2

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
Fields are just subsets of an HTML document, they should not have all of the json. Your styles and source code should go into `form.j2`